draft-ietf-tls-tls13.md

```diff
@@ -2301,14 +2303,22 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+Legacy algorithms
+: Indicates algorithms which are being deprecated because they use
+  SHA-1 but are still permitted for backward compatibility,
```

Comment:
Hmm, looks like we are losing the 2119-SHOULD NOT be used, which I would prefer to retain.